tests/core/vm/test_computation.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+
+from evm.vm import (
+    Computation,
+    Message,
+)
+
+from eth_utils import (
+    to_canonical_address,
+)
+
+from evm.exceptions import (
+    ValidationError,
+)
+
+CANONICAL_ADDRESS_1 = to_canonical_address("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6")
```

Comment:
This is a somewhat nit-picky naming convention thing, but, I've traditionally used the convention `thing_a` and `thing_b` over `thing_1` and `thing_2`. The reasoning is related to a bit of ocd with 0-indexed values and my brain thinking there should also be a `thing_0` as well as feeling like numeric ordering is much more heavily categorized as sequential (1 comes before 2), than alpha ordering.  With letters, there is less of a tendency for me at least to mentally line them up in a sequence.  They are just a set of labeled *things* without strict ordering.  Not sure if that makes sense and you don't need to update this if this approach doesn't hold water for you.